app/lib/navigation/state_file_id_question_navigation.rb

```diff
@@ -4,7 +4,7 @@ class StateFileIdQuestionNavigation < Navigation::StateFileBaseQuestionNavigatio
 
     SECTIONS = [
       Navigation::NavigationSection.new("state_file.navigation.section_1", [
-        # eligibility
+        Navigation::NavigationStep.new(StateFile::Questions::IdEligibilityResidenceController)
```

Comment:
I think we'll probably have to include these two pages in the navigation as well:
```
Navigation::NavigationStep.new(StateFile::Questions::EligibilityOffboardingController, false),
    Navigation::NavigationStep.new(StateFile::Questions::EligibleController),
```